Email Export when Done Button Update

xbryanc opened the pull request:
Button should now be more apparent and hovering it gives a description of why/how it is useful to the user
The email sent when export is finished will also contain domain name to give more context to the user

@dannyroberts

xbryanc committed:
add more info to exports email and change appearance of button

Button should now be more apparent and hovering it gives a description of why/how it is useful to the user

xbryanc committed:
fix max length line

thanks sticklerbot

dannyroberts commented:
@xbryanc could we get a screenshot or two here?

dannyroberts commented on corehq/apps/export/templates/export/partial/export_download_progress.html:
```diff
@@ -56,11 +56,14 @@
                         <button type="button"
                                 class="btn btn-default btn-full-width"
+                                title="For exports that may take a while to prepare"
+                                ng-style="{'background-color':'rgb(0, 180, 150)'}"
```
We usually try to stick to predefined button colors/appearances; you can pick from these: https://getbootstrap.com/docs/3.3/css/#buttons-options.

@orangejenny do you have a suggestion for button color for the email me when done button that will show up after clicking "Prepare Download" while the download is still loading? I believe at this point the `btn-primary` "Prepare Download" button is disabled but still on-screen.

orangejenny replied:
I'd just make it `btn-default`

dannyroberts replied:
Yeah, sorry that's what it originally was, and there was UAT feedback that the button was not prominent enough. Maybe it'd help to wait for a screenshot?

orangejenny replied:
got it - yeah, screenshot will help

xbryanc replied:
Sorry for the delay, I couldn't get this part of the server up yesterday for some reason.
![download exports picture](https://user-images.githubusercontent.com/35230409/35627543-2f777578-0667-11e8-8e5f-ee70e01f00ba.PNG)

orangejenny replied:
offlined with Bryan - he's going to have a look at making the "Collecting Data" button switch to `btn-default` while it's disabled and making the email button `btn-primary`, with the email button then switching to `btn-default` when it's clicked

xbryanc replied:
^that's what the latest updates change does, I re-explained in the commit statement as well

xbryanc committed:
change formatting of email exports button

the disabled loading button is now always default (it never did anything but display anyways)
the email exports button is btn-primary until clicked, then it turns into btn-default

orangejenny commented on corehq/apps/export/templates/export/partial/export_download_progress.html:
```diff
@@ -55,12 +55,14 @@
                         {% if request|toggle_enabled:"EMAIL_EXPORT_WHEN_DONE_BUTTON" %}
                         <button type="button"
-                                class="btn btn-default btn-full-width"
+                                title="For exports that may take a while to prepare"
```
Can you translate this?

xbryanc committed:
translate button title text

Hover information now translated

dannyroberts reviewed (approved):
Very nice, looks good with these changes. Thanks @xbryanc and @orangejenny!
